Gemfile

```diff
@@ -46,7 +46,7 @@ gem 'language_list', '~> 1.1.0'
 gem 'mail', '~> 2.7'
 gem 'maskedinput-rails'
 gem 'money-rails', '~> 1.10.0'
-gem 'mongo', '~> 2.6'
+gem 'mongo', '2.19.3'
```

Comment:
I think I told you un-pin mongo, this caused to downgrade from 2.20 to 2.19

Reply:
last time when I commented I am unable to start rails server/console. Let me try again.